MovieQuiz/Presentation/ AlertPresenter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//   AlertPresenter.swift
+//  MovieQuiz
+//
+//  Created by Alexander Farizanov on 04.12.2022.
+//
+import UIKit
+
+class AlertPresenter {
+    private weak var viewController : UIViewController?
+    init(viewController: UIViewController?)
+    {
```

Comment:
Давай везде вернем такой формат (он более привычен в среде iOS), так пишут шарписты :)  
```suggestion
    init(viewController: UIViewController?) {
```